cpp/src/parquet/encryption/internal_file_decryptor.h

```diff
@@ -76,12 +76,17 @@ class InternalFileDecryptor {
 
   ::arrow::MemoryPool* pool() { return pool_; }
 
+  /// This object must not outlive the InternalFileDecryptor.
   std::shared_ptr<Decryptor> GetFooterDecryptor();
+  /// This object must not outlive the InternalFileDecryptor.
   std::shared_ptr<Decryptor> GetFooterDecryptorForColumnMeta(const std::string& aad = "");
+  /// This object must not outlive the InternalFileDecryptor.
   std::shared_ptr<Decryptor> GetFooterDecryptorForColumnData(const std::string& aad = "");
+  /// This object must not outlive the InternalFileDecryptor.
   std::shared_ptr<Decryptor> GetColumnMetaDecryptor(
       const std::string& column_path, const std::string& column_key_metadata,
       const std::string& aad = "");
+      /// This object must not outlive the InternalFileDecryptor.
```

Comment:
fmt?

Reply:
Sorry, I didn't notice that comment. Will fix in next commit if has.